projects/core/src/cart/store/reducers/multi-cart.reducer.ts

```diff
@@ -42,6 +42,8 @@ export function cartEntitiesReducer(
   switch (action.type) {
     case CartActions.LOAD_MULTI_CART_SUCCESS:
     case CartActions.CREATE_MULTI_CART_SUCCESS:
+    case CartActions.CREATE_WISH_LIST_SUCCESS:
+    case CartActions.LOAD_WISH_LIST_SUCCESS:
       return action.payload.cart;
```

Comment:
what this reducer does?

Reply:
This reducer add the content of the wish list (cart) to the entities array that contains all the loaded carts